app/assets/stylesheets/application.css.scss

```diff
@@ -127,23 +119,11 @@ $image-path: 'img';
 
   dd {
     display: inline;
-    margin: 0
+    margin: 0;
   }
 
   dd::after {
     content: '';
     display: block;
   }
 }
-
-// Search overrides based on identity-site
-
-.usa-search-input--joined-right {
-  border-top-right-radius: 0;
-  border-bottom-right-radius: 0;
-  max-width: 12rem;
-}
-
-.text-heavy {
-  font-weight: 800;
-}
```

Comment:
I see a few references to this class. Do those need to be updated? I could imagine `text-bold` could work as a close, standard alternative.

https://github.com/18F/identity-dashboard/blob/1a6a438a8be5f442e478786eaca2ec0bd5480707/app/views/tools/saml_request.html.erb#L45-L55

https://github.com/18F/identity-dashboard/blob/1a6a438a8be5f442e478786eaca2ec0bd5480707/config/locales/service_providers.en.yml#L4-L5

Reply:
`text-heavy` is a [uswds utility class](https://designsystem.digital.gov/utilities/text-styles/)! i double checked the spots it's used to make sure it was available in the version that's in this version of the design system, so i think it's ok.